app/scripts/filters/resources.js

```diff
@@ -1084,12 +1089,16 @@ angular.module('openshiftConsole')
       DeploymentConfig: 'dc',
       ImageStream: 'is',
       Pod: 'pod',
+      ReplicaSet: 'rs',
       ReplicationController: 'rc',
       Route: 'route',
       Service: 'svc',
     };
     return function(kind) {
-      return abbreviated[kind] || kind;
+      if (!kind) {
+        return kind;
+      }
+      return abbreviated[kind] || kind.toLowerCase();
```

Comment:
kind.toLowerCase is inconsistent with what the CLI displays in similar scenarios, it should really be printing the resource, e.g. if it cant abbreviate something it should say deployments not deployment.  

where are we using abbreviateKind today?